### PR TITLE
Add preset for office=courier

### PR DESCRIPTION
**Pull request by govvin**
Add preset for office=courier, to improve tagging consistensy, and assist contributors in finding the tag for a [courier delivery service office](https://wiki.openstreetmap.org/wiki/Tag:office%3Dcourier).

Taginfo shows https://taginfo.openstreetmap.org/keys/office#values (search for "courier", "delivery") several similar values that appear to be trying to tag the same thing.

### CODE REVIEW

File: data/presets/office/courier.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "terms": [
+        "courier",
+        "delivery",
+        "express delivery",
+        "delivery service",
+        "parcel",
+        "package",
+        "messenger",
+        "carrier"
+    ],
```
Comment (tyrasd) [inline data/presets/office/courier.json:30]: could you please sort these alphabetically?

File: data/presets/office/courier.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "fields": [
+        "name",
+        "address",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "air_conditioning",
+        "branch_brand",
+        "brand"
```
Comment (tyrasd) [inline data/presets/office/courier.json:12]: I'd say, for courier services, the _brand_ information is quite important. Maybe we should promote at least the `brand` field to a regular `field`?

**Review by tyrasd — COMMENTED** [review]
Thanks! Looks good in general :+1: but please note my two minor comments below.

File: data/presets/office/courier.json (hunk 1 above)
Comment (govvin) [reply data/presets/office/courier.json:30]: Fixed. Thank you for the feedback.

File: data/presets/office/courier.json (hunk 2 above)
Comment (govvin) [reply data/presets/office/courier.json:12]: I agree with you! I'll fix that.

**Comment by govvin** [comment]
> Thanks! Looks good in general +1 but please note my two minor comments below.

Thank you for your feedback @tyrasd !  I've incorporated them in the PR now.

**Comment by 1ec5** [comment]
Has there been any discussion about this tag beyond this wiki page? I agree that it’s somewhat incongruous to tag delivery services as post offices, as you pointed out in osmlab/name-suggestion-index#8290, but I’m concerned that existing usage of this tag is actually a mix of post office–like delivery services and [headquarters offices of bicycle messenger services](https://wiki.openstreetmap.org/wiki/Talk:Tag:office%3Dcourier#Bicycle_couriers).

**Comment by govvin** [comment]
> Has there been any discussion about this tag beyond this wiki page? I agree that it’s somewhat incongruous to tag delivery services as post offices, as you pointed out in [osmlab/name-suggestion-index#8290](https://github.com/osmlab/name-suggestion-index/pull/8290), but I’m concerned that existing usage of this tag is actually a mix of post office–like delivery services and [headquarters offices of bicycle messenger services](https://wiki.openstreetmap.org/wiki/Talk:Tag:office%3Dcourier#Bicycle_couriers).

Hi @1ec5 ! Thanks for checking this out. 

Many of the discussions (forums, mailing list) I've seen had to to with the original proposal, but limiting myself to what is written on the current wiki page, I suppose it meets the definition?

I have a slightly longer reply wiki talk link you posted, since that's  easily accessible to more OSM contributors.

**Comment by 1ec5** [comment]
I hadn’t noticed that this wiki page was based on [an abandoned proposal](https://wiki.openstreetmap.org/wiki/Proposal:Office%3Dcourier), so I guess that’s something. To be clear, I don’t have any objection to splitting out a new tag for a superset of POIs that’s less specific than `amenity=post_office`; mapping these facilities as post offices has been bothering me since I first starting doing that. I was just surprised to see no mention of the tag on the tagging list, given all the activity around “[postal partners](https://wiki.openstreetmap.org/wiki/Proposal:Shop_as_post-partner)” a couple years ago.

For what it’s worth, these same companies often place [drop-off boxes](https://wiki.openstreetmap.org/wiki/Key:post_box:type#United_States) right next to the postal service’s official mail collection box. These are currently being tagged as `amenity=post_box`, by analogy with `amenity=post_office`.

**Pull request closed without merging**